yaml/_dumper.ts

```diff
@@ -601,8 +601,7 @@ function writeScalar(
 function writeFlowSequence(
   state: DumperState,
   level: number,
-  // deno-lint-ignore no-explicit-any
-  object: any,
+  object: unknown[],
```

Comment:
Now that this is a `DumperState` method, and `state.dump` is the only value ever passed as `object`, why not just remove this last argument and use `this.dump` within this method?